x-pack/plugins/maps/public/shared/layers/sources/es_search_source/es_search_source.js

```diff
@@ -186,8 +186,9 @@ export class ESSearchSource extends AbstractESSource {
       if (!field) {
         return;
       }
-
-      filteredProperties[propertyName] = field.format.convert(filteredProperties[propertyName]);
+      const converter = field.format.getConverterFor('html');
```

Comment:
Can we rename `converter ` to `htmlConverter`? That will make the `?` operator make more sense since the fall back is the format convert method so a converter still exists